README.md

```diff
@@ -8,11 +8,12 @@ The Stan Math Library is licensed under the [new BSD license](LICENSE.md).
 
```

Comment:
Can you add a quick line under the Licensing section stating that Stan Math includes an Apache-2 licensed dependency and that anyone using Stan Math is fine as long as they dont redistribute unary binaries licensed GPL-2 and "see more in the wiki".

Reply:
Good idea.